test/Stats.unittest.js

```diff
@@ -134,8 +134,10 @@ describe(
 			});
 		});
 		describe("toJson", () => {
-			it("returns plain object representation", () => {
-				const mockStats = new Stats({
+			let fakeCompilation;
+			beforeEach(() => {
+				
```

Comment:
```suggestion
```

Reply:
@ooflorent Good catch. I've pushed an update with all of these fixes